### ISSUE TITLE
Caliptra Synthesis Errors

### ISSUE THREAD

**Issue by amullick007** [chipsalliance/caliptra-rtl#317]
We are getting below synthesis errors with latest RTL.

```
Warning: In design 'el2_ifu_mem_ctl__449949_NV_caliptra', input pin 'din[6]' of hierarchical cell 'perr_dat_ff' has one or more internal loads, but is not connected to any nets. 'Logic 0' is assumed. (LINT-59)
Warning: In design 'el2_ifu_mem_ctl__449949_NV_caliptra', input pin 'din[5]' of hierarchical cell 'perr_dat_ff' has one or more internal loads, but is not connected to any nets. 'Logic 0' is assumed. (LINT-59)
Warning: In design 'el2_ifu_mem_ctl__449949_NV_caliptra', input pin 'din[4]' of hierarchical cell 'perr_dat_ff' has one or more internal loads, but is not connected to any nets. 'Logic 0' is assumed. (LINT-59)
Warning: In design 'el2_ifu_mem_ctl__449949_NV_caliptra', input pin 'din[3]' of hierarchical cell 'perr_dat_ff' has one or more internal loads, but is not connected to any nets. 'Logic 0' is assumed. (LINT-59)
Warning: In design 'el2_ifu_mem_ctl__449949_NV_caliptra', input pin 'din[2]' of hierarchical cell 'perr_dat_ff' has one or more internal loads, but is not connected to any nets. 'Logic 0' is assumed. (LINT-59)
Warning: In design 'el2_ifu_mem_ctl__449949_NV_caliptra', input pin 'din[1]' of hierarchical cell 'perr_dat_ff' has one or more internal loads, but is not connected to any nets. 'Logic 0' is assumed. (LINT-59)
Warning: In design 'el2_ifu_mem_ctl__449949_NV_caliptra', input pin 'din[0]' of hierarchical cell 'perr_dat_ff' has one or more internal loads, but is not connected to any nets. 'Logic 0' is assumed. (LINT-59)
```

Upon checking the RTL, I saw that _perf_dat_ff.din[6:0]_ is connected to _ifu_ic_rw_int_addr_ff[12:6]_. The driver of _ifu_ic_rw_int_addr_ff_ is coming from below module code which is disabled because _pt.ICACHE_ENABLE_ is 0.

```
         rvdffie #(pt.ICACHE_TAG_LO-pt.ICACHE_TAG_INDEX_LO+pt.ICACHE_NUM_WAYS+1) tag_addr_ff (.*,
                                                                                              .clk(free_l2clk),
                                                                                              .din({ifu_ic_rw_int_addr_w_debug[pt.ICACHE_INDEX_HI:pt.ICACHE_TAG_INDEX_LO],
                                                                                                    ifu_tag_wren_w_debug[pt.ICACHE_NUM_WAYS-1:0],
                                                                                                    ic_valid_w_debug}),
                                                                                              .dout({ifu_ic_rw_int_addr_ff[pt.ICACHE_INDEX_HI:pt.ICACHE_TAG_INDEX_LO],
                                                                                                     ifu_tag_wren_ff[pt.ICACHE_NUM_WAYS-1:0],
                                                                                                     ic_valid_ff})
                                                                                              );

```

Is it ok for synthesis to treat the input _din_ of _perr_dat_ff_ as 0?

**Comment by Nitsirks** [comment]
Yes, it has no loads that I can find with icache disabled.

We should see have Veer maintainers move this flop to the icache section also.

**Comment by amullick007** [comment]
@Nitsirks Is this issue resolved in RTL1.0 release?

**Comment by Nitsirks** [comment]
No. Looks like there is a PR out for VeeR that will fix it. I assume we'll absorb this for 1.1

https://github.com/chipsalliance/Cores-VeeR-EL2/issues/149

**Comment by amullick007** [comment]
Thanks Michael, do you know the approximate timeline for 1.1 release?

**Comment by bharatpillilli** [comment]
@Nitsirks - close this?

**Comment by calebofearth** [comment]
Fixed for 1.1 release in 020cdc6 via #483
